README.md

```diff
@@ -460,6 +473,318 @@ const user = await UserFactory.buildAsync();
 - Hooks are executed in the order they were registered
 - Multiple hooks of the same type can be chained
 
+## JSON Schema Integration
+
+Interface-Forge supports creating factories directly from JSON Schema definitions, making it perfect for teams that already use JSON Schema, OpenAPI/Swagger specifications, or need language-agnostic schema definitions.
+
+### JSON Schema Installation
+
+The JSON Schema integration is available as an optional feature to keep the core library lightweight:
+
+```shell
+# Install the core library
+npm install --save-dev interface-forge
+
+# Install optional JSON Schema dependencies
+npm install --save-dev ajv ajv-formats
+```
+
+### JSON Schema Basic Usage
+
+Create factories from JSON Schema definitions:
+
+```typescript
+import { createFactoryFromJsonSchema } from 'interface-forge/json-schema';
```

Comment:
What versions of json Schema are supported? 

Please add a comprehensive test for each draft version that is supported.

We should support from draft-7 onwards probably